### DISCUSSION TITLE
Exporting to HTML5 in the roadmap?

### DISCUSSION THREAD

**Discussion by GeneralVimes** [airsdk/Adobe-Runtime-Support#2095] [General]
I remember once it was mentioned that it might be cool to export an AIR project as an HTML5 game (to let AS3 developers make Web games again like in good old times :) )

Is such a feature in a roadmap? I understand that now instead of adding something new the team has to cope with everchanging regulations from Google, Apple and other platform holders, and this takes much time. So, no urging you @ajwfrost , just wondering

**Comment by ajwfrost** [comment]
It's a topic that comes up for discussion from time to time! There are some complexities around licensing rights for this sort of functionality though - I'm hoping that these will become clearer as this year passes so we should know more where we stand at the start of next year whether we can add this kind of web target. Definitely something we'd like to do... but equally, it's a big task!

thanks
